addons/voice/org.openhab.voice.voicerss/src/main/java/org/openhab/voice/voicerss/internal/VoiceRSSTTSService.java

```diff
@@ -164,22 +164,35 @@ public AudioStream synthesize(String text, Voice voice, AudioFormat requestedFor
         return audioFormats;
     }
 
-    /**
-     * Up to now only MP3 supported.
-     */
-    private final AudioFormat getAudioFormat(String format) {
-        // MP3 format
-        if (AudioFormat.CODEC_MP3.equals(format)) {
-            // we use by default: MP3, 44khz_16bit_mono
-            Boolean bigEndian = null; // not used here
-            Integer bitDepth = 16;
-            Integer bitRate = null;
-            Long frequency = 44000L;
+    private final AudioFormat getAudioFormat(String apiFormat) {
+        Boolean bigEndian = null;
+        Integer bitDepth = 16;
+        Integer bitRate = null;
+        Long frequency = 44000L;
```

Comment:
I see that this wasn't introduced by you, but isn't the normal frequency 44.1 KHz and not 44.0 KHz?